runtime/vm/CRIUHelpers.cpp

```diff
@@ -171,15 +173,16 @@ findinstanceFieldOffsetHelper(J9VMThread *currentThread, J9Class *instanceType,
  * @param[in] currentThread vmThread token
  * @param[in] userData J9InternalHookRecord pointer
  *
- * @return BOOLEAN TRUE if no error, otherwise FALSE
+ * @return const char* NULL if no error, otherwise a NLS messsage
```

Comment:
I prefer you pass it in by reference, in the future we may need to return other things

Reply:
Agreed, changed it to by reference.